src/bindings/python/src/pyopenvino/core/offline_transformations.cpp

```diff
@@ -33,13 +34,16 @@ void regmodule_offline_transformations(py::module m) {
 
     m_offline_transformations.def(
         "apply_moc_transformations",
-        [](std::shared_ptr<ov::Model> model, bool cf) {
+        [](std::shared_ptr<ov::Model> model, bool cf, bool smart_reshape = false) {
```

Comment:
```suggestion
        [](std::shared_ptr<ov::Model> model, bool cf, bool smart_reshape) {
```
It needs to be moved to `py::args` section, example:
https://github.com/openvinotoolkit/openvino/blob/378e334ee6dd3507e219f92c9ab9672d1a9321fc/src/bindings/python/src/pyopenvino/core/tensor.cpp#L19-L23